File: sdk/typescript/src/___PLAYGROUND___/networks/get-networks.ts
import { begin } from '../helpers';

export const getNetwork = async () => {
  let context = await begin();

  let networks = await context.client.network.get(31337);
  console.log('networks', networks);

  await context.end();
};

getNetwork().then((_) => () => console.log('get-network done'));

File: sdk/python/src/__PLAYGROUND__/networks/get-networks.py
import asyncio

from __PLAYGROUND__.helper import begin, end


async def getNetwork():
    try:
        client, _, _, rrelayer_node = await begin()

        networks = await client.network.get(31337)
        print("networks", networks)
    except Exception as e:
        print("getNetwork failed:", e)
    finally:
        end(rrelayer_node)


if __name__ == "__main__":
    asyncio.run(getNetwork())
    print("get-network done")
